app/src/datasets/chorus_dt/places_resolution.py

```diff
@@ -53,6 +54,29 @@ def compute_distance_between_points(lon0, lat0, lon1, lat1):
 
     return c * r
 
+def calc_CO2(trips:pandas.DataFrame,
+             CO2_frame:pandas.DataFrame):
+    
+    short_plane = (trips['prestation'] == 'A') & (trips['distance'] <= 1000)
+    long_plane = (trips['prestation'] == 'A') & (trips['distance'] > 1000)
+    train = (trips['prestation'] == 'T')
+    commun = (trips['prestation'] == 'TC')
+    
+    CO2_short_plane = CO2_frame.loc[(CO2_frame['Prestation'] == 'A') &
```

Comment:
I am wondering if it would not simplify a lot the code and the workflows to have these 4 values in a dict global variable somewhere.

It seems a lot of overhead and could be replaced by something like 
```
CO2_short_plane = CO2_COSTS['short_plane']
```

Reply:
Are these values going to be used elsewhere? Isn't it kind of overkill to create a dict from a csv array specifically for this one function?

Reply:
Though it would be prettier....

Reply:
What am I saying is that you could get ride of the CSV, and just have the dictionary declared in the code somewhere.